token-service/src/main/java/org/p2p/token/service/manager/TokenServiceEventManager.kt

```diff
@@ -0,0 +1,18 @@
+package org.p2p.token.service.manager
+
+class TokenServiceEventManager {
```

Comment:
TokenServiceEventManager -> TokenServiceEventNotifier / TokenServiceEventObservable

Reply:
I would've even rename it to `TokenServiceEventListenerDispatcher` because it actually dispatches listeners and their data. But in the name of shortness I agree with the name `TokenServiceEventManager`